userspace/plugin/plugin_types.h

```diff
@@ -78,6 +78,12 @@ typedef struct ss_plugin_event
 	uint64_t ts;
 } ss_plugin_event;
 
+struct const_sized_buffer {
```

Comment:
I would rename this `ss_plugin_byte_buffer`. From a naming perspective, the size can actually be changed at runtime. The `ss_plugin` prefix is just something we have in common with all the other types.